### ISSUE TITLE
Main Navigation is broken on some devices

### ISSUE THREAD

**Issue by ryanswedal** [chriscoyier/CSS-Tricks-v10#45]
Main navigation is unusable on a HTC Inspire running Android v2.3.3 with HTC Sense v2.1. A couple of menu items are shown but tiny and hard to click. Other items are hidden from view. The "navigation 'n' search" button is not there at all.

**Comment by Blackhawkso** [comment]
Same thing happens for a brief moment on iPhone but then after a second or so it shows

**Comment by speicus** [comment]
This can be reproduced in Opera Mobile Emulator.
And in the actual Opera Mobile on Android 4.0, HTC Incredible S.
Chrome, though, displays everything correctly on same device.

**Comment by chriscoyier** [comment]
Just pushed a fix for this. Reopen this thread if it's still messed up but I think I got it.

**Comment by ryanswedal** [comment]
It's still wonky. The "navigation 'n' search" shows up but the home and forum text/icons appear on top of it. Then when clicked the first 4 menu items appear on the left as they should but the last 4 items are not shown.

Just out of curiosity what droid version are you previewing this with?

**Comment by chriscoyier** [comment]
Can confirm in browserstack -- http://cl.ly/JH3c

**Comment by chriscoyier** [comment]
think I fixed again

**Comment by speicus** [comment]
It helped, but not by much. Now I can see "Navigation 'N' Search" all right, but it does not look like a button.
http://img696.imageshack.us/img696/232/csstricksmenu.png

If I click on it's center, then menu shows up.
But if I click on letter "c" in "Search", then I end up in the forum!

Really, Chris, install a Opera Mobile Emulator and see for yourself.

There is also another issue. You can't open menu until the page is fully loaded. And if the page is really heavy (like the v10 discussion) — it takes forever to load. I literally have to go back to front page to use menu.

And the animation looks so awful on mobile... All those twitches and little jumps. And really, my phone isn't that bad (HTC Incredible S).

I wish you'd just leave menu visible on mobile devices, as it was in previous version. :)

**Comment by chriscoyier** [comment]
I think there are some more tweaks I can make to make it better.

If I left the menu visible on mobile, the vast majority of the top of the page would be navigation and even more people would be unhappy. This, I'm pretty sure of =)
